README.md

```diff
@@ -120,6 +121,42 @@ job.getQueryResults().on('data', function(row) {});
 ```
 
 
+## Google Cloud Bigtable
+
+- [API Documentation][gcloud-bigtable-docs]
+- [Official Documentation][cloud-bigtable-docs]
+
+*You may need to [create a cluster][gcloud-bigtable-cluster] to use the Google Cloud Bigtable API with your project.*
+
+#### Preview
+
+```js
+var gcloud = require('gcloud');
+
+// Authenticating on a per-API-basis. You don't need to do this if you auth on a
+// global basis (see Authentication section above).
+var bigtable = gcloud.bigtable({
+  projectId: 'my-project',
+  keyFilename: '/path/to/keyfile.json',
+  zone: 'my-zone',
```

Comment:
v2 API will change this. (eta - very soon) no Zone or cluster, replaced by instanceId